File: Python/Algorithms/BubbleSort.py
def bubbleSort(ar, n):
    for x in range(n - 1):
        for y in range(n - x - 1):
            if ar[y] > ar[y + 1]:
                ar[y], ar[y+1] = ar[y+1], ar[y]


# Test Code

n = int(input())

ar = list(map(int, input().split()))

bubbleSort(ar, n)

print(ar)

File: CPP/Algorithms/bubbleSort.cpp
#include <iostream>
using namespace std;

void bubbleSort(int arr[], int n)
{
    int i, j, temp = 0;
    for (i = 0; i < n - 1; i++)
    {
        for (j = 0; j < n - i - 1; j++)
        {
            if (arr[j] > arr[j + 1])
            {
                temp = arr[j + 1];
                arr[j + 1] = arr[j];
                arr[j] = temp;
            }
        }
    }
}

// Test Code

int main()
{
    int i, n, ar[100];
    cin >> n;
    for (i = 0; i < n; i++)
    {
        cin >> ar[i];
    }
    bubbleSort(ar, n);
    for (int i = 0; i < n; i++)
        cout << ar[i] << ' ';
    return 0;
}
